packages/cosmic-swingset/test/run-policy.test.ts

```diff
@@ -1,53 +1,60 @@
 /* eslint-env node */
-import test from 'ava';
-import { assert, q, Fail } from '@endo/errors';
-import { E } from '@endo/far';
+import type { ParamsSDKType } from '@agoric/cosmic-proto/swingset/swingset.js';
 import { BridgeId, deepCopyJsonable, objectMap } from '@agoric/internal';
+import type { BlockInfo } from '@agoric/internal/src/chain-utils.js';
 import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
-import {
-  defaultBootstrapMessage,
-  defaultInitMessage,
-  makeCosmicSwingsetTestKit,
-} from '../tools/test-kit.js';
+import type { SwingSetConfigDescriptor } from '@agoric/swingset-vat';
+import { assert, Fail, q } from '@endo/errors';
+import { E } from '@endo/far';
+import type { TestFn } from 'ava';
+import anyTest from 'ava';
+import type { KVStore } from '../src/helpers/bufferedStorage.js';
 import { provideEnhancedKVStore } from '../src/helpers/bufferedStorage.js';
 import {
   DEFAULT_SIM_SWINGSET_PARAMS,
   makeVatCleanupBudgetFromKeywords,
+  type VatCleanupKeywordsRecord,
 } from '../src/sim-params.js';
+import {
+  defaultBootstrapMessage,
+  defaultInitMessage,
+  makeCosmicSwingsetTestKit,
+} from '../tools/test-kit.js';
 
-/** @import { ManagerType, SwingSetConfig } from '@agoric/swingset-vat' */
-/** @import { KVStore } from '../src/helpers/bufferedStorage.js' */
+const test = anyTest as TestFn;
 
 /**
  * Converts a Record<name, specifierString> into Record<name, { sourceSpec }>
  * for defining e.g. a `bundles` group.
  *
- * @param {Record<string, string>} src
- * @returns {import('@agoric/swingset-vat').SwingSetConfigDescriptor}
+ * @param src
```

Comment:
This could go away with your new JSDoc in `*.ts` eslint that makes `@param` optional?  Do you want to remove this now, or after the eslint config change?